data/sql/updates/pending_db_world/rev_1624283834038071800.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1624283834038071800');
+
+UPDATE `creature_template` SET `unit_flags` = 131076, `flags_extra` = 8388866 WHERE (`entry` = 17680);
```

Comment:
https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits

Reply:
@FrancescoBorzi you should optimize keira for this :D